interfaces/IAirplay.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "Module.h"
+
+namespace WPEFramework {
+    namespace Exchange {
+
+        struct EXTERNAL IAirplay : virtual public Core::IUnknown {
+            enum { ID = ID_AIRPLAY};
+
+            virtual ~IAirplay() { }
+
+            enum state : uint16_t {
+                PLAYING = 0x0001,
+                STOPPED = 0x0002,
+                SUSPENDING = 0x0004
+            };
+
+            struct EXTERNAL INotification : virtual public Core::IUnknown {
+                  enum { ID = ID_AIRPLAY_NOTIFICATION };
+                  virtual void StateChange(const IAirplay::state state) = 0;
+                  virtual void Exit(const uint32_t exitCode) = 0;
+                  virtual void NotifyClient(const string& eventName) = 0;
```

Comment:
Repetition of naming in an INotification interface we do expect Notify...s to an interested party. Assumption is made here with the name that those interested must be Clients, are Servers not allowed to subscribe to these events ;-) , So would it not be more descriptive to say here _void Event(const string& eventName) = 0;_

Reply:
Ack.